pkg/koordlet/statesinformer/api.go

```diff
@@ -16,16 +16,70 @@ limitations under the License.
 
 package statesinformer
 
-import corev1 "k8s.io/api/core/v1"
+import (
+	topov1alpha1 "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/apis/topology/v1alpha1"
+	topologyclientset "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/clientset/versioned"
+	slov1alpha1 "github.com/koordinator-sh/koordinator/apis/slo/v1alpha1"
+	koordclientset "github.com/koordinator-sh/koordinator/pkg/client/clientset/versioned"
+	schedv1alpha1 "github.com/koordinator-sh/koordinator/pkg/client/clientset/versioned/typed/scheduling/v1alpha1"
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/metriccache"
+	corev1 "k8s.io/api/core/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+type InformerGetter interface {
+	GetInformer(name informerName) Informer
+}
+
+type Informer interface {
+	Run(stopCh <-chan struct{}) error
+	HasSynced() bool
+
+	GetNode() *corev1.Node
```

Comment:
perhaps the generic Informer does not require the following methods `GetNode`, `GetNodeSLO`, ... `GetContainerByPodUID`